components/script/dom/htmlinputelement.rs

```diff
@@ -432,6 +490,35 @@ impl HTMLInputElementMethods for HTMLInputElement {
     // https://html.spec.whatwg.org/multipage/#dom-input-alt
     make_setter!(SetAlt, "alt");
 
+    // https://html.spec.whatwg.org/multipage/#attr-fe-autocomplete
```

Comment:
This link is about the *content* attribute, not the IDL one. See the [spec](https://html.spec.whatwg.org/multipage/#dom-fe-autocomplete) for the IDL attribute.